test/withStyles_test.jsx

```diff
@@ -570,7 +570,7 @@ describe('withStyles', () => {
         const MockComponent = ({ styles, css }) => <div {...css(styles.primary)} />;
         MockComponent.propTypes = { ...withStylesPropTypes };
         const StyledComponent = withStyles(stylesFn)(MockComponent);
-        const wrapper = shallow(<StyledComponent />);
+        const wrapper = mount(<StyledComponent />);
```

Comment:
why is this change needed?

Reply:
Not super sure actually. I commented about it here https://github.com/airbnb/react-with-styles/pull/240#issuecomment-589309710

Reply:
ah - you'd need to add a `.dive()` here to invoke the deeper level (one dive per HOC/wrapper)